build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,38 +291,14 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
             {
                 Command.Create(Path.Combine(pkgDir, "pack.cmd"))
```

Comment:
I didn't dig deeply so I wasn't sure if Exec was different from Command.Create or not, I will look into making this one code path.